### DISCUSSION TITLE
DAP draft

### DISCUSSION THREAD

**Discussion by ap1987-ap** [munoztd0/DAP_2021#19] [General]
Hi David @munoztd0 
Quick question. It states that we can upload a draft version of our DAP till today 31st.
You mentioned, that we can send a draft to you till the 4th of january. Is that correct?
Because Im not really finished right now and the 4th would definitly be better for me.

Thanks for your answer, Best
Anja

**Comment by munoztd0** [comment]
Yes that what I said and it's still true.
